src/paperbot/api/routes/auth.py

```
@@ -5,7 +5,13 @@
 from typing import Optional
 
 from fastapi import APIRouter, HTTPException, Depends
-from pydantic import BaseModel, EmailStr
+from pydantic import BaseModel
+
+try:
+    import email_validator  # noqa: F401
+    from pydantic import EmailStr
+except Exception:  # pragma: no cover - exercised only in minimal test envs
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)